<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StatisticalTable</class>
 <widget class="QWidget" name="StatisticalTable">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>762</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>10</y>
     <width>141</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>统计报表</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>540</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="salesWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>90</y>
     <width>291</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QWidget" name="salesChart" native="true">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>361</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
